scripts/script2.r

```diff
@@ -0,0 +1,143 @@
+# Assignment 1:  
+library(tweedie) 
+library(ggplot2)
+
+simTweedieTest <-  
+  function(N){ 
+    t.test( 
+      rtweedie(N, mu=10000, phi=100, power=1.9), 
+      mu=10000 
+    )$p.value 
+  } 
+
+
+# Assignment 2:  
+MTweedieTests <-  
+  function(N,M,sig){ 
+    sum(replicate(M,simTweedieTest(N)) < sig)/M 
+  } 
+
+
+# Assignment 3:  
+df <-  
+  expand.grid( 
+    N = c(10,100,1000,5000, 10000), 
+    M = 1000, 
+    share_reject = NA) 
+
+
+
+    # Incorporating doParallel for the execution of MTweedieTests within the loop
```

Comment:
You should not have an indent for the code below. Having indents in the code implies that the code is inside a loop, function etc.